RHEL/6/transforms/shorthand2xccdf.xslt

```diff
@@ -19,6 +19,10 @@
 <xsl:variable name="ovalfile">unlinked-rhel6-oval.xml</xsl:variable>
 <xsl:variable name="defaultseverity" select="'low'" />
 
+<!-- Specify location of official DISA FSO Zip benchmark (RHEL-6 specific URI) -->
+<!-- Fix for issue https://github.com/OpenSCAP/scap-security-guide/issues/1035 -->
+<xsl:variable name="disa_fso_rhel6_uri">http://iasecontent.disa.mil/stigs/zip/Jan2016/U_RedHat_6_V1R10_STIG.zip</xsl:variable>
```

Comment:
As FSO increments their releases, this URL will get old. Should we link to `````http://iase.disa.mil/stigs/os/unix-linux/Pages/index.aspx````` instead?